src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
```

Comment:
`<a href="#" class="nav__link is-active">Apple</a>` is quite difficult to read. You should really care about how easy is reading of your code, so my suggestion for improvement is to move each attribute to the new line + move the text to a separate one 
```suggestion
          <li class="nav__item">
            <a 
              href="#"
              class="nav__link is-active"
            >
              Apple
            </a>
          </li>
```